git-hg/git-hg.sh

```diff
@@ -16,41 +16,41 @@
 git clone https://github.com/frej/fast-export.git
 hg clone http://hg.openjdk.java.net/jdk8u/jdk8u/ openjdk_hg
 
-cd openjdk_hg
+cd openjdk_hg || exit
 # get latest mercurial tag
-hgTag=`hg log -r "." --template "{latesttag}\n"`
+#hgTag=`hg log -r "." --template "{latesttag}\n"`
 
 bash ./get_source.sh
 for i in corba jaxp jaxws langtools jdk hotspot nashorn; do
    echo "cleaning up $i"
    # removes the mercurial stuff from the additional repos
-   cd $i
+   cd $i || exit
    rm -rf .hg .hgignore .hgtags
-   cd -
+   cd - || exit
 done
-cd $WORKSPACE
+cd "$WORKSPACE" || exit
 
 mkdir openjdk_git
-cd openjdk_git
+cd openjdk_git || exit
 git init
 # convert mercurial to github
-$WORKSPACE/fast-export/hg-fast-export.sh -r $WORKSPACE/openjdk_hg
+"$WORKSPACE"/fast-export/hg-fast-export.sh -r "$WORKSPACE"/openjdk_hg
 for i in corba jaxp jaxws langtools jdk hotspot nashorn; do
 	# copy the additional repos in
-	cp -r $WORKSPACE/openjdk_hg/$i $i
+	cp -r "$WORKSPACE"/openjdk_hg/$i $i
 done
 git checkout
 # remove remaining mercurial stuff
 rm -rf .hg .hgignore .hgtags README-builds.html README get_source.sh
 # create new README
-cp $WORKSPACE/openjdk-build/git-hg/README.md .
+cp "$WORKSPACE"/openjdk-build/git-hg/README.md .
```

Comment:
same here as above - this will be in line with the guideline on https://github.com/AdoptOpenJDK/openjdk-build/pull/54